wren-launcher/utils/docker.go

```diff
@@ -101,7 +141,7 @@ func PrepareDockerFiles(openaiApiKey string, port int, projectDir string) error
 	}
 
 	// replace the content with regex
-	newEnvFileContent := replaceEnvFileContent(string(envFileContent), openaiApiKey, port)
+	newEnvFileContent := replaceEnvFileContent(string(envFileContent), openaiApiKey, port, pg_pwd)
```

Comment:
I saw you pass the pg_pwd to env, but didn't see the pg_username

Reply:
oh I see

Reply:
addressed in 79e1f6d